projects.go

```diff
@@ -930,3 +930,59 @@ func (s *ProjectsService) ListProjectForks(pid interface{}, opt *ListProjectsOpt
 
 	return forks, resp, err
 }
+
+// ListRunners gets a list of runners accessible by the authenticated user.
+//
+// GitLab API docs: https://docs.gitlab.com/ce/api/runners.html
+func (s *ProjectsService) ListRunners(pid interface{}, opt *ListRunnersOptions, options ...OptionFunc) ([]*Runner, *Response, error) {
+	project, err := parseID(pid)
+	if err != nil {
+		return nil, nil, err
+	}
+	u := fmt.Sprintf("projects/%s/runners", url.QueryEscape(project))
+
+	req, err := s.client.NewRequest("GET", u, opt, options)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	var p []*Runner
+	resp, err := s.client.Do(req, &p)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return p, resp, err
+}
+
+// EnableRunnerOptions represents the available EnableRunner() options.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ce/api/runners.html#enable-a-runner-in-project
+type EnableRunnerOptions struct {
+	RunnerID int `json:"runner_id"`
+}
+
+// EnableRunner enables an available specific runner in the project.
+//
+// GitLab API docs: https://docs.gitlab.com/ce/api/runners.html
+func (s *ProjectsService) EnableRunner(pid interface{}, opt *EnableRunnerOptions, options ...OptionFunc) (*Runner, *Response, error) {
+	project, err := parseID(pid)
+	if err != nil {
+		return nil, nil, err
+	}
+	u := fmt.Sprintf("projects/%s/runners", url.QueryEscape(project))
+
+	req, err := s.client.NewRequest("POST", u, opt, options)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	var p *Runner
+	resp, err := s.client.Do(req, &p)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return p, resp, err
+}
```

Comment:
I fully understand that you put those methods here, but for I would like to follow the docs as much as possible. This is done in other situations like this (were some methods could be in multiple places) as well, so that it remains very clear for people where to find which methods (by just finding them in the docs to see in which API they are defined).

This means that I think these calls should be called `ListProjectRunner` and `EnableProjectRunner` and be defined on the `runner` service instead...